telefunc/plugin/next/index.ts

```diff
@@ -0,0 +1,25 @@
+import type { NextConfig } from 'next'
+import { resolve } from 'path'
+
+export default plugin
+
+function plugin(nextConfig: NextConfig = {}): NextConfig {
+  return Object.assign({}, nextConfig, {
+    webpack: (config, options) => {
+      config.module.rules.push({
+        test: /\.telefunc\./,
```

Comment:
(We could do the test more robust with `/\.telefunc\.[a-zA-Z0-9]+/` but I guess it's fine we can leave it like that.)